coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/core/ProposalDevelopmentControllerBase.java

```diff
@@ -255,11 +255,12 @@ public ModelAndView save(ProposalDevelopmentDocumentForm form) throws Exception
          ((ProposalDevelopmentViewHelperServiceImpl)form.getViewHelperService()).setOrdinalPosition(form.getDevelopmentProposal().getProposalPersons());
          saveAnswerHeaders(form, form.getPageId());
 
+         getTransactionalDocumentControllerService().save(form);
```

Comment:
shouldn't we being running the audit rules before we save, or does it make a difference in this case?